tests/scalar.rs

```diff
@@ -0,0 +1,18 @@
+use momento::requests::cache::basic::set::Set;
+use momento_test_util::CACHE_TEST_STATE;
+use uuid::Uuid;
+
+#[tokio::test]
+async fn set_happy_path() {
+    let client = CACHE_TEST_STATE.client.clone();
+    let cache_name = CACHE_TEST_STATE.cache_name.clone();
```

Comment:
do you need to clone these? if so it could be a code smell in our API, we want to try to minimize the places that people need to clone things. We should be cloning things ourselves when it's necessary so that callers don't have to.

Reply:
Cloning was the pattern that was in `cache_sorted_set` but TIL borrowing would work fine too:
```
let client = &CACHE_TEST_STATE.client;
let cache_name = &CACHE_TEST_STATE.cache_name;
```